src/_assets/css/_code_shared.scss

```diff
@@ -70,13 +70,12 @@ pre.console-output {
   .p { @extend .pun; } /* Punctuation */
   .c, .c1, .cm { @extend .com; } /* Comment */
   .cp { @extend .typ; } /* Comment.Preproc */
-  .l, .m, .mi, .mf, .mh, .mx, .mb, .mo { @extend .lit; } /* Literal.* */
```

Comment:
Did you mean to omit .mh?

Otherwise, lgtm!

Reply:
Oops, thanks!